Client/mods/deathmatch/logic/CPacketHandler.cpp

```diff
@@ -3964,6 +3964,13 @@ void CPacketHandler::Packet_EntityAdd(NetBitStreamInterface& bitStream)
                                 pPolygon->AddPoint(vertex.data.vecPosition);
                             }
                             pEntity = pShape = pPolygon;
+                            if (bitStream.Can(eBitStreamVersion::SetPolygonHeight))
```

Comment:
Better clarification.
```suggestion
                            if (bitStream.Can(eBitStreamVersion::SetColPolygonHeight))
```